v2/README.md

```diff
@@ -59,14 +59,19 @@ It's important for renters and hosts to upgrade before **June 6th** so that they
 
 #### ...Siacoin holders?
 
-First, to be absolutely clear: Your Siacoins are cryptographically controlled by your seed, and Sia v2 doesn't change that. `walletd` is backwards-compatible with the existing blockchain data, so you don't need to move your Siacoins in order to prevent them from getting "lost". That said, Sia v2 software is deprecating support for old seeds and, instead, supports a new industry standard type of address that enables more flexible multisig scenarios and atomic swaps.
+Your Siacoins remain secured by your seed and don’t change with Sia v2. `walletd` is fully compatible with the existing blockchain data, so you're not required to move your coins to prevent loss. However, Sia v2 is deprecating support for legacy 28- and 29-word seeds in favor of a new industry-standard address format that supports more flexible multisig and atomic swaps.
```

Comment:
The seed phrase has nothing to do with addresses. 

`in favor of industry standard BIP39 12-word seed phrases.`